packages/app-info/README.md

```diff
@@ -11,7 +11,8 @@ A utility to get application information (e.g. the system code) in a consistent
     * [`appInfo.releaseVersion`](#appinforeleaseversion)
     * [`appInfo.systemCode`](#appinfosystemcode)
     * [`appInfo.processType`](#appinfoprocesstype)
-    * [`appInfo.cloudProvider](#appinfocloudprovider)
```

Comment:
**issue:** looks like we deleted the cloud provider docs link by accident.

Reply:
oops nice spot. Have re-added it now